mail-service.js

```diff
@@ -0,0 +1,35 @@
+const path = require("path");
+
+const FIELDS_WITH_MULTILINE_SUPPORT = ["bcc", "to", "cc"];
+
+const NEW_LINE_WITH_OPTIONAL_COMMA_REGEX = /(?:,\s*)?\n/g;
+const COMMA_WITH_WHITESPACE_REGEX = /\s*,\s*/g;
+
+function sendWithTransport(transporter, {
+  attachmentPaths,
+  ...mailOptions
```

Comment:
List all the options here. This will help to understand what options are expected by the transport.